### ISSUE TITLE
Storybook failing to start in a fresh react app - failure in stats.toJson

### ISSUE THREAD

**Issue by abhagupta** [storybookjs/storybook#9752]
Getting a bug while installation and setup.
```/node_modules/@storybook/core/dist/server/dev-server.js:80
          stats.toJson(options.errorDetails);
                ^

TypeError: Cannot read property 'toJson' of null
```
Node v10.15.3
NPM 6.4.1
storybook/react v5.3.11

**To Reproduce**
1. Created a simple react app using npx create-react-app my-app
2. Followed instructions from storybook start guide https://storybook.js.org/docs/guides/quick-start-guide/

Tried multiple times.  Btw.. this is manual installation of storybook

**Comment by juliocymatic** [comment]
I tried to run storybook for Vue but got same error, I need help please

node v12.13.0
npm 6.13.0
@storybook/core@5.3.11

```
$ /node_modules/@storybook/core/dist/server/dev-server.js:80
          stats.toJson(options.errorDetails);
                ^

TypeError: Cannot read property 'toJson' of null
```

**Comment by shilman** [comment]
Closing as dupe to #9749 

Fixed in 5.3.12 AFAIK - waiting for verification
